examples/distributed_lock/example.py

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+from dapr.clients import DaprClient
+import logging
+import warnings
+
+
+def main():
+    # Lock parameters
+    store_name = 'lockstore'  # as defined in components/lockstore.yaml
+    resource_id = 'example-lock-resource'
+    client_id = 'example-client-id'
+    expiry_in_seconds = 60
+
+    with DaprClient() as dapr:
+        logging.info(f'Will try to acquire a lock from "{store_name}" for resource={resource_id}')
```

Comment:
A little bit more friendly and Python IMHO is using Print statements, and %s for string interpolation even if it is the "old style".

e.g.
    print('Will try to acquire a lock from %s for resource=%s' % (store_name, resource_id), flush=True)

Reply:
I remember that there was a mix of `logging` and `print` in the tutorials and for some reason I found the output from the ones using `logging` better.

But the intent for this file is to act as an small sample code showing how to use the API. Using `logging` is just adding noise for no added benefit. Ditto for `f''` strings. I will convert it to `prints` with plain string interpolation.